icu4c/source/test/intltest/listformattertest.cpp

```diff
@@ -611,6 +611,55 @@ void ListFormatterTest::TestBadStylesFail() {
     }
 }
 
+void ListFormatterTest::TestCreateStyled() {
+    IcuTestErrorCode status(*this, "TestCreateStyled");
+    // Locale en has interesting data
+    struct TestCase {
+        const char* locale;
+        UListFormatterType type;
+        UListFormatterWidth width;
+        const char16_t* expected;
+    } cases[] = {
```

Comment:
Please add more tests to cover
1. where there are only 1 item in the list
2. where there are only 2 items in the list

Reply:
Done.